ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Node.hs

```diff
@@ -14,10 +14,7 @@
 
 module Ouroboros.Consensus.Shelley.Node (
     MaxMajorProtVer (..)
-  , ProtocolParamsAllegra (..)
-  , ProtocolParamsAlonzo (..)
-  , ProtocolParamsMary (..)
-  , ProtocolParamsShelley (..)
+  , ProtocolParams (..)
```

Comment:
INFO: protocol parameters for all shelley eras are now exported by this module. It should now be sufficient import `O.C.Shelley.Node` instead of a combination of `O.C.Shelley.Node`, `O.C.Shelley.Node.Praos` and `O.C.Shelley.Node.TPraos`.